data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,143 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
```

Comment:
add `library(janitor)` here. Great new [package](https://github.com/sfirke/janitor) for data cleaning